patches/server/1048-Brigadier-based-command-API.patch

```diff
@@ -635,11 +635,9 @@ index 0000000000000000000000000000000000000000..b7f1c15800f175760717fcea38a7256a
 +                .executes(redirectTo.getCommand())
 +                .requires(redirectTo.getRequirement())
 +                .build();
-+            for (final CommandNode<CommandSourceStack> child : redirectTo.getChildren()) {
-+                redirect.addChild(child);
-+            }
```

Comment:
Good catch, though I believe it needs to be re-added in the other branch of the if-statement. But you could also drop the if-statement and just do
```java
final LiteralArgumentBuilder<CommandSourceStack> builder = Commands.literal(aliasLiteral)
    .executes(redirectTo.getCommand())
    .redirect(redirectTo)
    .requires(redirectTo.getRequirement())
for (final CommandNode<CommandSourceStack> child : redirectTo.getChildren()) {
    builder.then(child);
}

return this.registerIntoDispatcher(new PluginCommandNode(aliasLiteral, plugin, builder.build(), description), override);
```